Dockerfile

```diff
@@ -1,7 +1,7 @@
 #================================
 # Stage 1: Build Gitbase
 #================================
-FROM golang:1.12-alpine as builder
+FROM golang:1.11-alpine as builder
```

Comment:
this is wrong, isn't it?

Reply:
I've copied the `Dockerfile` version from master. It uses 1.11.